fastly/s3.go

```diff
@@ -9,12 +9,24 @@ import (
 
 type S3Redundancy string
 type S3ServerSideEncryption string
+type S3AccessControlList string
 
 const (
-	S3RedundancyStandard      S3Redundancy           = "standard"
-	S3RedundancyReduced       S3Redundancy           = "reduced_redundancy"
+	S3RedundancyStandard   S3Redundancy = "standard"
+	S3RedundancyReduced    S3Redundancy = "reduced_redundancy"
+	S3RedundancyOneZoneIA  S3Redundancy = "onezone_ia"
+	S3RedundancyStandardIA S3Redundancy = "standard_ia"
+
 	S3ServerSideEncryptionAES S3ServerSideEncryption = "AES256"
 	S3ServerSideEncryptionKMS S3ServerSideEncryption = "aws:kms"
+
+	S3AccessControlListPrivate                S3AccessControlList = "private"
+	S3AccessControlListPublicRead             S3AccessControlList = "public-read"
+	S3AccessControlListPublicReadWrite        S3AccessControlList = "public-read-write"
+	S3AccessControlListAWSExecRead            S3AccessControlList = "aws-exec-read"
+	S3AccessControlListAuthenticatedRead      S3AccessControlList = "authenticated-read"
+	S3AccessControlListBucketOwnerRead        S3AccessControlList = "bucket-owner-read"
+	S3AccessControlListBucketOwnerFullControl S3AccessControlList = "bucket-owner-full-control"
```

Comment:
There looks to be another value `log-delivery-write` defined in https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl.

I presume that extra value was omitted because the Fastly UI didn't make reference to it (which makes sense).

Reply:
Yes the main reason was because it wasn't mentioned in the UI, but also I guessed that it wasn't relevant anyway as it's a bucket-only policy (as opposed to object-level) and, as far as I'm aware, is only used for S3's own access logs (but I may be wrong)

Reply:
Also actually if you look at the `x-amz-acl` [request parameter](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateMultipartUpload.html#API_CreateMultipartUpload_RequestParameters) then it's not specified in that list either, so I think that confirms the previous comment